<!-- DOCTYPE -->
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <!-- Viewport Meta Tag -->
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>
      COMPARE INVENTORIES DATABASE
    </title>
    <!-- Bootstrap -->
    <link rel="stylesheet" type="text/css" href="/assets/css/bootstrap.min.css">
    <!-- Main Style -->
    <link rel="stylesheet" type="text/css" href="/assets/css/main.css">
    <!-- Responsive Style -->
    <link rel="stylesheet" type="text/css" href="/assets/css/responsive.css">
    <!--Fonts-->
    <link rel="stylesheet" media="screen" href="/assets/fonts/font-awesome/font-awesome.min.css">
    <link rel="stylesheet" media="screen" href="/assets/fonts/simple-line-icons.css">
	  <link rel="stylesheet" media="screen" href="/assets/fonts/fonts.css">
  	<link rel="stylesheet" type="text/css" href="/assets/css/dataTables.bootstrap4.min.css">

    <!-- Color CSS Styles  -->
    <link rel="stylesheet" type="text/css" href="/assets/css/colors/green.css" media="screen" />
    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js">
    </script>
    <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js">
    </script>
    <![endif]-->
  </head>
  <body>
	<div class="container-fluid">
  <!-- Header area wrapper starts -->
    <header id="header-wrap">

      <!-- Roof area starts -->

      <!-- Roof area Ends -->

      <!-- Header area starts -->
      <section id="header">

        <!-- Navbar Starts -->
        <nav class="navbar navbar-light" data-spy="affix" data-offset-top="50">
            <button class='navbar-toggler hidden-md-up pull-xs-right' data-target='#main-menu' data-toggle='collapse' type='button'>
              ?
            </button>
            <!-- Brand -->
            <a class="navbar-brand" href="/index.html">
              <img src="/assets/img/Logo_Compare_Small.png" alt="Compare Logo" style="width: 200px;">
            </a>
            <div class="collapse navbar-toggleable-sm" id="main-menu">
              <!-- Navbar Starts -->
              <ul class="nav nav-inline">
                <li class="nav-item dropdown">
                  <a class="nav-link" href="/index.html" role="button" aria-haspopup="true" aria-expanded="false">Home</a>
                </li>
                <!--
                <li class="nav-item dropdown">
                  <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="protocols" role="button" aria-haspopup="true" aria-expanded="false">
                    Protocols
                  </a>
                  <div class="dropdown-menu">
                    <a class="dropdown-item" href="/protocols/animal_human.html">Human & Animal</a>
                    <a class="dropdown-item" href="/protocols/food.html">Food</a>
                    <a class="dropdown-item" href="/protocols/transport.html">Transport</a>
                    <a class="dropdown-item" href="/protocols/vector.html">Vector</a>
                  </div>
                </li>
              -->
              <li class="nav-item dropdown">
                  <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="/modelling/frameworks.html" role="button" aria-haspopup="true" aria-expanded="false">
                    Model Frameworks
                  </a>
                  <div class="dropdown-menu">
                    <a class="dropdown-item" href="/modelling/frameworks.html">Open-source Frameworks</a>
                  </div>
                </li>
                <li class="nav-item dropdown">
                  <a class="nav-link dropdown-toggle active" data-toggle="dropdown" href="/pipelines/pipelines.html" role="button" aria-haspopup="true" aria-expanded="false">
                    Metagenomics Pipelines
                  </a>
                  <div class="dropdown-menu">
                    <a class="dropdown-item active" href="/pipelines/pipelines.html">Metagenomics Pipelines</a>
                  </div>
                </li>
                <li class="nav-item dropdown pull-md-right">
                  <a class="nav-link open-search" href="#" role="button" aria-haspopup="true" aria-expanded="false"><span class="top_search_icon"><i class="icon-magnifier"></i></span></a>
                </li>
              </ul>
            </div>

        </nav>
        <!-- Navbar Ends -->

      </section>
    </header>
    <!-- Header-wrap Section End -->

    <!-- Service Block-1 Section -->
    <section id="service-block-main" class="split section">
      <!-- Container Starts -->

        <!--<div class="row">
          &lt;!&ndash; Form for navbar search area &ndash;&gt;
          <div class="col-sm-12">
          <form class="full-search" id="searchform" action="test.php" method="get">
            <div class="container">
              <input type="text" name="searchtext" placeholder="Type to Search">
              <button type="submit" form="searchform" class="btn btn-common btn-lg">Search</button>
            </div>
          </form>
          </div>
        </div>-->
		<div class="row">
			<div class="col-sm-12">
				<h1>Inventory of existing metagenomics pipelines</h1>
				<table id="example" class="table table-striped table-responsive">
				</table>
			</div>
		</div>
	</section>
</div>
<script src="/assets/js/jquery-min.js"></script>
    <!-- Bootstrap JS -->
    <script src="/assets/js/bootstrap.min.js"></script>
     <!-- All JS plugin Triggers -->
    <script src="/assets/js/main.js"></script>
    <!--<script src="assets/js/color-switcher.js"></script>-->
	<script src="/assets/js/datatables.js"></script>
	<script src="/assets/js/dataTables.bootstrap4.min.js"></script>
    <script type="text/javascript" charset="utf8">
		var dataSet = [

["Name","Authors","Publication date","Doi","Goal","Unique feature","Limitation","Target organism(s)","Method","Search algorithm","Sequence type","DB (classification)","Filter algorithm","DB (filter)","Classification level","Can detect novel/unknown viruses?","Steps ","GUI?","Visualizes results?","Runtime","Accepts datatypes","Tested w/ read length","Raw reads/contigs","HPC or PC?","Freely available?","Licence","User manual","Computer platform","Sequencing platform","Reproducibility","Link (software)","Other tools/pipelines tested","Tested by (tools)","Independent benchmarks","Described by","Validation","Accuracy","Up-to-date","Citations (Scopus)","Software used","Citations (Google Scholar, 2017-03-28)","Title of publication"],
["FACS","Stranneheim et al.","13/05/2010","10.1093/bioinformatics/btq230","read classification/filter","Bloom filters","best suited to removal of known sequences","any","similarity (exact match k-mers)","Bloom filter","nt","custom","-","-","species","limited","S","no","no","~20 times faster than BLAT/SSAHA2","?","269 average","reads","?","yes","MIT","https://github.com/SciLifeLab/facs","Linux","454","high","https://github.com/SciLifeLab/facs","BLAT, SSAHA2","-","-","-","-","98-100%","yes (last commit 17 Dec 2015)","22","Perl, C","39","Classification of DNA sequences using Bloom filters"],
["NBC","Rosen et al.","08/11/2010","10.1093/bioinformatics/btq619","classification of metagenomic reads","easy to use webserver for phylegenetic analysis, novice and expert settings, classifies every read","long runtimes?, classifies everything - potentially high FPR","any","composition (N-mer frequency profiles)","custom","nt","unique N-mer frequency profiles of 635 microbial genomes","-","-","species","limited","S","yes (webservice)","no","~ 21 h","fasta (compressed possible)","230 average","reads","webserver/PC","yes","-","http://nbc.ece.drexel.edu/tutorial.php","Any (webservice)","Any","medium-high (medium for local installation, high for webserver)","http://nbc.ece.drexel.edu/","MG-RAST, Galaxy, CAMERA, WebCARMA, PhyloPythia","Kraken, Taxy-Pro, MetLab","Garcia-Etxebarria et al. (2014)","-","-","100% classified (no positive/negative assessment), 78-97% (Kraken), 2-48% (MetLab, species-kindgom)","no (no date provided on website, publication from 2011)","60","Perl, C++","125","NBC: the Naive Bayes Classification tool webserver for taxonomic classification of metagenomic reads"],
["ProViDE","Ghosh et al.","26/03/2011","PMCID: PMC3082859","accurate classification of viral reads from metagenomes","rank-specific classification criteria","slow, and only viruses","virus","similarity","BLASTx","aa","NCBI nr","-","-","species","no","S","no","no","hours (1hr/100,000 reads - slower than MEGAN)","Sanger, 454","100-800","reads","?","academic, non-profit","custom","-","Linux (Ubuntu/Fedora)","454, Sanger","medium","http://metagenomics.atc.tcs.com/binning/ProViDE/","MEGAN","MetLab","-","Fancello et al. (2012)","-","better than MEGAN, 25-44% (MetLab, species-kingdom)","no (last update on site: 2013)","?","Perl, Python","19","ProViDE: A software tool for accurate estimation of viral diversity in metagenomic samples"],
["PathSeq","Kostic et al.","01/05/2011","10.1038/nbt.1868","discover microorganisms in human tissue","good filter for human sequences","computationally expensive (compared to most other NGS analyses), limited to databases","pathogens","similarity","BLAST","nt, aa","NCBI viruses, fungi, bacterial and archaeal, and NCBI nr","MAQ, Megablast, BLASTN","1000 Genomes Project female reference, Ensembl Homo sapiens cDNA, human genome and transcriptome BLAST database (NCBI), 3 assembled human genomes from NCBI, NCBI Homo sapiens RNA database, Ensembl human genome reference","species","yes","PFFAS","no","no","?","?","76, 100, 101 bp","reads","HPC","yes","MIT (modified)","-","Linux, cloud (Amazon EC2, Apache Hadoop)","Any","medium-high (high in cloud, medium for self-installation)","https://www.broadinstitute.org/software/pathseq/","-","READSCAN, RINS","-","-","-","?, up to 86% (Readscan), 99% (RINS)","no (only date on website is copyright 2010)","91","Python, Java, C++, C, Hadoop, MAQ, Megablast, Blast, Velvet, RepeatMasker","158","PathSeq: software to identify or discover microbes by deep sequencing of human tissue"],
["SRSA","Isakov et al.","11/06/2011","10.1093/bioinformatics/btr349","pathogen identification using short RNA subtraction and assembly","short RNA","short RNA from human host","human pathogens","similarity","Megablast","nt","All non-redundant GenBank CDS translations+PDB+SwissProt+PIR+PRF'","BWA","NCBI hg19","species","no","PFAS","no","no","?","all types of current sequencing platforms","34 average","reads","?","upon request","-","-","Linux","Illumina","medium","-","-","-","-","-","-","?","no","?","BWA, fastx_toolkit, BLAST, MegaBlast, Velvet","40","Pathogen detection using short-RNA deep sequencing subtraction and assembly"],
["MEGAN4","Huson et al.","20/06/2011","10.1101/gr.120618.111 ","taxonomic and functional annotation of metagenomics reads","LCA and visualization","needs mapping program (if not DIAMOND, it's probably a slow one)","metagenomes","similarity","BLAST","nt, aa","NCBI nt/nr","-","-","species","limited","S","yes","yes","?","?","35-800 bp","reads/contigs","?","for academic use","-","http://ab.inf.uni-tuebingen.de/software/megan4/","Any","Any","high","http://ab.inf.uni-tuebingen.de/software/megan4/","-","ProViDE, Taxy-Pro, MetLab","Peabody et al. (2015), Lindgreen et al. (2016), Bazinet & Cummings (2012), Randle-Boggis et al. (2016)","RIEMS, Fancello et al. (2012)","-","22-100% (short to long reads), 56-85% (MetLab, species-kindgom), ~15-70% (Peabody, sensitivity BLASTN-precisionRAPSearch2), 43% (Lindgreen, MCC), 43-100% (Bazinet & Cummings, sensitivity-precision), 98.6% (Randle-Boggis, sensitivity)","no (version 6 is out)","?","Java","752","Integrative analysis of environmental sequences using MEGAN4"],
["VMGAP","Lorenzi et al.","01/07/2011","10.4056/sigs.1694706","functional annotation of viral metagenomes","hiearchical protein annotation","runs only at JCVI","virus","similarity","BLAST, HMMER","aa","GenBank NRDB, GenBank environmental ENV_NT & ENV_NR, UniProtDB, OMNIOMEDB, PFAM, TIGRFAM HMMs, ACLAME HMMs, GenBank CDDDB, pfam2gomappingsDB","-","-","protein","no","SSSSS-pp","no","no (possible to export to Excel or METAREP)","?","protein fasta (translated predicted ORFs)","no","contigs","HPC (grid)","no (only at JCVI)","-","-","?","?","n/a (high when at JCVI)","-","MG-RAST","-","-","Fancello et al. (2012)","-","more detailed protein annotations than MG-RAST","no","?","HMMER, BLAST (NCBI-toolkit), SignalP, TMHMM, PRIAM","25","TheViral MetaGenome Annotation Pipeline (VMGAP):an automated tool for the functional annotation of viral Metagenomic shotgun sequencing data"],
["Metavir","Roux et al.","01/09/2011","10.1093/bioinformatics/btr519","virome analysis","web server for comprehensive virome analysis","too detailed for diagnostics/clinical use(?)","virus","similarity + phylogeny","BLASTx, MUSCLE, PhyML, HMMER","nt, aa","RefSeq, PFAM, NCBI nr","-","-","monophyletic groups","no","SASPh","yes (webservice)","yes (tree)","interactive","?","100-400+","reads/contigs","40 CPU-HPC","superceded by newer version","-","http://metavir-meb.univ-bpclermont.fr/index.php?page=Tutorial","Any (webservice)","Any","high (webservice)","http://metavir-meb.univ-bpclermont.fr/","-","-","-","Fancello et al. (2012)","-","?","no (problems in 2015 seem unresolved)","48","MUSCLE, CAP3, Gblocks, PhyML, Scriptree","88","Metavir: a web server dedicated to virome analysis"],
["RINS","Bhaduri et al.","01/02/2012","10.1093/bioinformatics/bts100","identify non-human sequences","fast host filtering - low RAM requirement (for desktop use) - similar to PathSeq","installation of dependencies","non-human","similarity","BLAT, BLAST","nt","custom virus db(?)","Bowtie","human genome","family","no","SPFAS","no","no","~ 3 h (2CPU), ~15 min (16CPU)","fastq?","36-500 paired-end","reads","PC (2 CPU, 8 GM RAM)","yes","-","supplied with download(?)","Linux","Mate-paired 36-500 bp","low-medium","http://khavarilab.stanford.edu/tools-1/#tools","PathSeq","CaPSID, ClinicalPathoScope, Virana","-","RIEMS","PCR + Sanger sequencing","100%","no (website has moved, but is still online)","32","Blat, Bowtie, Trinity, BLAST","51","Rapid identification of non-human sequences in high-throughput sequencing datasets"],
["VIROME","Wommack et al.","30/07/2012","10.4056/sigs.2945050","viral metagenome analysis","extensive virome analysis","only viruses, mostly marine(?)","virus","similarity","BLAST","aa","UniRef 100 peptide, SEED, ACLAME, COG, GO, KEGG, MGOL, CAMERA, UniVec","BLASTn. tRNAscan-SE","a rRNA subject database","-","no","PFS","yes (webservice)","yes (charts, tables)","?","fasta & qual, fastq, sff","no","reads","HPC","use yes, download no","-","http://virome.dbi.udel.edu/, Tutorial videos","Any (webservice)","454","high","http://virome.dbi.udel.edu/app/#view=home","-","-","-","Fancello et al. (2012)","-","?","no (website is online, but no date is mentioned)","29","Adobe Flex, MySQL, Blast, tRNA scan SE, MetaGene Annotator, CD-Hit 454","59","VIROME: a standard operating procedure for analysis of viral metagenome sequences"],
["CaPSID","Borozan et al.","17/08/2012","10.1186/1471-2105-13-206","pathogen identification in human tumor genomes and transcriptomes","MongoDB with sequence DBs, results and metadata, for collaborative projects","made for tumors","pathogens","similarity","Novoalign/Bowtie2 (any)","nt","UCSC/NCBI GRCh37/hg19 + viral + microbial + fungal genomes, or custom from GenBank","same as search","UCSC/NCBI GRCh37/hg19 or custom","species","no","S","yes","yes (coverage maps, result tables)","~ 20 min","bam","65","reads","16 CPU, 5 GB (can run on desktop)","yes","GNU GPLv3","https://github.com/capsid/capsid/wiki","Linux, Mac OS","Any","medium-high","https://github.com/capsid/capsid","RINS","Virana","-","-","in vitro validation","100%, 2-100% (Virana, divergent to homologous sequences)","no (last update 2 Jun 2012)","?","Python 2.7, MongoDB, OpenJDK, BioPython, pysam, Novoalign, BioScope, JBrowse, Groovy-Grails","26","CaPSID: A bioinformatics platform for computational pathogen sequence identification in human genomes and transcriptomes"],
["VirusSeq","Chen et al.","17/11/2012","10.1093/bioinformatics/bts665","identify viral integration sites in human cancer tissue","MOSAIK aligner","only viruses in database","virus","similarity","MOSAIK","nt","GIB-V, hg19Virus (hg19 + TCGA cancer-associated viruses)","MOSAIK","hg19 (UCSC)","-","no","FS","no","no","> 1 week","fastq","paired-end reads","reads","HPC?","yes","-","-","Linux, Mac OS","Paired-end Illumina","low-medium","http://odin.mdacc.tmc.edu/~xsu1/VirusSeq.html","-","-","-","-","serology, colorimetric in situ hybridization, immunohistochemistry","?","no","24","Perl, MOSAIK","50","VirusSeq: software to identify viruses and their integration sites using next-generation sequencing of human cancer tissue"],
["READSCAN","Naeem et al.","28/11/2012","10.1093/bioinformatics/bts684","pathogen discovery","fast and scalable, portable to any computer type","limited to human viruses","pathogens","similarity","SMALT","nt","?","SMALT","?","species","no","S","no","no","< 27 min on 16 CPU-HPC - 4 h","Illumina/454/SOLiD","no","reads","portable from laptop to HPC-cluster","yes","GNU GPLv3","http://www.cbrc.kaust.edu.sa/readscan/, supplied with download - in scripts","Linux","Any","low-medium","http://cbrc.kaust.edu.sa/readscan/","RINS, PathSeq","Clinical PathoScope","-","RIEMS","-","up to 100%","no","18","Perl, SMALT, Makeflow","30","READSCAN: a fast and scalable pathogen discovery program with accurate genome relative abundance estimation"],
["ViralFusionSeq","Li et al.","01/01/2013","10.1093/bioinformatics/btt011","discover viral integration and resonstruct fusion transcripts at single-base resolution","viral integration site discovery and viral expression at single-base resolution","only viral integration","virus","similarity","BWA, BLAST","nt","viral sequences and human decoy sequences","-","-","-","no","PS","no","no","> 1 week","RNA-seq?","no","reads","HPC?","yes","GNU GPLv3","supplied with download","Linux","Illumina","low-medium","https://sourceforge.net/projects/viralfusionseq/","-","-","-","-","Sanger sequencing","?","no (website cannot be reached at the moment)","20","Perl, BWA, BLAST","31","ViralFusionSeq: accurately discover viral integration events and reconstruct fusion transcripts at single-base resolution"],
["Taxy-Pro","Klingenberg et al.","15/02/2013","10.1093/bioinformatics/btt077","more sensitive detection of under-represented organisms (Archaeae and viruses)","protein-profile based, two indices to assess quality","restricted to CDS, depends on (limited) Pfam, CoMet less sensitive than HMMER3, not suitable as stand-alone - needs complementation","metagenomes","similarity","CoMet","aa","Pfam + metagenomes","-","-","superkingdom, phylum(, class)","no","S","no","no","about three orders of magnitude faster than speed-optimized BLAST","fastq?","450 bp simulated 454 reads, public datasets (HMP, Oil Seep)","reads","PC","yes","-","-","Any (webservice or MATLAB)","454, Illumina","high","http://gobics.de/TaxyPro/","(NBC, PhymmBL,) BLASTN+LCA, MEGAN, MG-RAST, WebCARMA, (MetaPhyler,) CombiBLAST(, Taxy)","-","-","-","-","?","no (files last modified 2013)","?","MATLAB, CoMet webserver","14","Protein signature-based estimation of metagenomic abundances including all domains of life and viruses"],
["VirusFinder","Wang et al.","01/05/2013","10.1371/journal.pone.0064465","accurate identification of viral integration sites","accurate, robust integration site detectoin","only viruses in database","virus","similarity","BLAST, BLAT, Bowtie2, BWA","nt","RINS virus DB, or GIB-V","Bowtie2","either NCBI build 37/36 or UCSC hg19/hg18","-","no","FS(AS/S-pp)","no","no","3 d","fastq/bam","no","reads","8 CPU-HPC","yes","-","https://bioinfo.uth.edu/VirusFinder/VirusFinder-manual.pdf","Linux","Any","medium","http://bioinfo.mc.vanderbilt.edu/VirusFinder/","VirusSeq, ViralFusionSeq","-","-","VIP","-","?","yes","20","Perl, BLAST+, BLAT, Bowtie2, BWA, iCORN, CREST, GATK, SAMtools, SVDetect, Trinity","49","VirusFinder: Software for Efficient and Accurate Detection of Viruses and Their Integration Sites in Host Genomes through Next Generation Sequencing Data"],
["REGA typing tool v3","Pineda-Pe�a et al.","07/05/2013","10.1016/j.meegid.2013.04.032","HIV-1 genotyping for clinical and surveillance purposes","automated genotyping of HIV-1","slow, low reproducibility (of phylogenetic methods in general)","virus","similarity + phylogeny","BLAST, TreePuzzle","nt","custom","-","-","subtype","no","SPh","yes (webservice)","yes (table, trees)","500 seqs in 5h","fasta (preprocessed)","no","reads/contigs","webserver","yes","GNU GPLv2","http://regatools.med.kuleuven.be/typing/v3/hiv/typingtool/tutorial","Any (webservice)","Any","high","http://regatools.med.kuleuven.be/typing/v3/hiv/typingtool/","-","-","-","-","-","?","no","37","Php, Java, R, TreePuzzle","79","Automated subtyping of HIV-1 genetic sequences for clinical and surveillance purposes: Performance evaluation of the new REGA version 3 and seven other tools"],
["IMSA","Dimon et al.","23/05/2013","10.1371/journal.pone.0064546","filter search","action files","requires selection of mapping tool and databases","any","similarity","BLAST","nt","user-defined universal database, (NCBI nt?)","Bowtie, BLAT, BLAST","user-defined (human genome)","species","no","FS","no","yes (TaxMap)","hours","Illumina, 454","50 bp, 70 bp single-end, 150 bp paired-end","reads","scalable from desktop to SUN Grid cluster (tested with Grid)","yes","-","https://sourceforge.net/projects/arron-imsa/files/IMSA_UserManual_v2.pdf/download","Linux, Mac OS","454, Illumina","medium-high (Python scripts with few dependencies)","https://sourceforge.net/projects/arron-imsa/?source=directory","-","ClinicalPathoScope","-","-","-","86-91%","no (last update 17-4-2014)","7","Python, Blast, Bowtie2, Blat","10","IMSA: Integrated Metagenomic Sequence Analysis for Identification of Exogenous Reads in a Host Genomic Background"],
["LMAT","Ames et al.","15/09/2013","10.1093/bioinformatics/btt389","classification of sequencing reads as fast as they are produced by the sequencing machine","scale with supercomputer","very high RAM requirement and decreased sensitivity(?)","any","similarity (match k-mers)","custom","nt","NCBI microbial genomes","-","-","species","limited","S-pp","no","no","1.3 Mbp/s","Illumina?","36-200","reads","HPC (high RAM)","yes","-","-","Linux","Any","low","https://sourceforge.net/projects/lmat/","PhymmBL, MetaPhlAn, Genometa","ClinicalPathoScope","Lindgreen et al. (2016)","-","-",">99% (bacteria), 30% (Lindgreen, MCC)","yes (last update: 7-5-2015)","24","gcc, Python, OpenMP, MPI","64","Scalable metagenomic taxonomy classification using a reference genome database"],
["VirusHunter","Zhao et al.","01/10/2013","10.1371/journal.pone.0078470","virus discovery","novel virus identification","focused on 454","virus","similarity","BLAST","nt, aa","NCBI nt/nr","BLASTn","host genome, default: human (NCBI 1000 genomes?)","species","yes","PFS(S)","no","no","?","454","no","reads","HPC","non-profit","register","-","Linux","454","low","https://www.ibridgenetwork.org/#!/profiles/9055559575893/innovations/103/","-","-","-","-","EM, serology (hemagglutanation inhibition)","?","no (website only states Launched: April 2012)","28","Perl, MySQL, Blast, CD-HIT, RepeatMasker","46","Identification of Novel Viruses Using VirusHunter -- an Automated Data Analysis Pipeline"],
["Virana","Schelhorn et al.","03/10/2013","10.1371/journal.pcbi.1003228","detect viral transcripts in human cancers","finding lowly abundant viruses and chimeric reads, and highly divergent viral reads","lower sensitivity than other tools when viral divergence is low","virus","similarity + phylogeny","STAR(, BLAST, LASTZ)","nt","NCBI reference virus genomes, Repbase endogenous retroviruses","STAR","UCSC human genome (hg19, GRCh37), Ensembl human cDNA","species","limited","PS(A)(S)Ph","no","no","~ 30 min/CPU","fastq?","36, 54, 95, 100","reads","PC","yes","Apache licence","https://github.com/schelhorn/virana","Linux, Mac OS","Paired-end Illumina","medium-high (self-install may be difficult, but Galaxy should run with no problem","https://github.com/schelhorn/virana","CaPSID, RINS","-","-","-","FISH, Southern blot","14-88% (divergent to more homologous sequences)","no (last commit 2 years ago: July 2014)","7","Python, STAR, BWA-mem, LASTZ, RazerS3, Jalview(, plumbum, numpy, biopython, pysam, ftputil, HTSeq, matplotlib, bx-python)","9","Sensitive Detection of Viral Transcripts in Human Tumor Transcriptomes"],
["QuasQ","Poh et al.","05/12/2013","10.1186/1471-2105-14-355","genome recovery, quasispecies estimation","quasispecies identification","454 sequencing, viruses with good references","virus","similarity","Bowtie2","nt","the reference genome","-","-","quasispecies","no","PS-pp","no","no","?","454","330 average","reads","?","yes","-","http://www.statgen.nus.edu.sg/~software/quasq.html","Linux, Mac OS","454","low (separate dependencies, tarball)","http://www.statgen.nus.edu.sg/~software/quasq.html","ShoRAH, ViSpA, PredictHaplo, QuRe, QuasiRecomb","-","-","-","-","98-100%","no (Last updated 10 Jul 2014)","0","Perl, Bowtie2","5","Viral quasispecies inference from 454 pyrosequencing"],
["MePIC","Takeuchi et al.","01/01/2014","10.7883/yoken.67.62","pathogen identification for clinical specimens","easy cloud-pipeline for dummies, counts human hits but does not filter them","limited by DBs, tailored for clinical use","pathogens","similarity","Megablast","nt","NCBI nt","BWA","UCSC/NCBI hg19","species","no","PFS","yes (webservice)","no","10 h on 1 CPU, 6 min on 100 CPUs (Megablast only)","fastq","200(?)","reads","HPC","use yes, download upon request","-","https://mepic.nih.go.jp/mepic/manual/","Any (webservice)","Illumina","high (web-based)","https://mepic.nih.go.jp/","-","-","-","-","-","?","no (website does not work with local Firefox, which is v45 (newest as of 18-04-2016))","6","fastq-mcf (ea-utils), BWA, Megablast","15","MePIC, Metagenomic Pathogen Identification for Clinical Specimens"],
["ClassyFlu","Van der Auwera et al.","03/01/2014","10.1371/journal.pone.0084558","classification (typing) of Influenza A","profile HMMs for Influenza A typing","only Influenza A","Influenza A","similarity","HMMER","nt","custom (from NCBI Influenza Virus Resource)","-","-","subtype","no","S","yes (webservice)","no","?","fasta","1.7 kb","contigs","webservice","yes","GNU GPLv3","supplied with download","Any (webservice), or Linux, Mac OS","Any","high (web), medium-high (local?)","http://bioinf.uni-greifswald.de/ClassyFlu/query/init","IRD-CT, manual BLAST + phylogeny","-","-","-","-","up to 97% concordance with gold standard (IRD-CT)","no (last modified 24 Sep 2013)","?","HMMER","0","ClassyFlu: Classification of Influenza A Viruses with Discriminatively Trained Profile-HMMs"],
["Kraken","Wood & Salzberg","03/03/2014","10.1186/gb-2014-15-3-r46","metagenomic classification","very fast, possible to combine with other tools or use as host filter","need much RAM","metagenomes (mostly bacteria)","similarity (exact match k-mers)","Kraken","nt","NCBI RefSeq microbial, or GenBank genomes for bacteria and archaea","-","-","Genus, species","limited","S-pp","no (Illumina BaseSpace integration?)","no","~ 1 h","fasta (fastq with special switch)","no","reads","HPC","yes","GNU GPLv3","http://ccb.jhu.edu/software/kraken/MANUAL.html","Linux","Any","high","https://github.com/DerrickWood/kraken","MegaBlast, PhymmBL, NBC, MetaPhlAn","MetaShot, IMSA+A, Taxonomer, Kaiju, DUDes, RIEMS, SMART","Peabody et al. (2015), Lindgreen et al. (2016)","-","-","73-91%, 93-99% (MetaShot), 23% (IMSA+A, many false positive bacteria), 66-93% (Taxonomer), 30-95% (Kaiju, sensitivity@genus-precision@phylum), 83-86% (DUDes), 42% (GOTTCHA, many FPs), 16-100% (RIEMS), 30-85% (SMART, species-class), 88-95% (MetLab, species-kingdom), <90% (Peabody, precision, sensitivity very low), 80% (Lindgreen, MCC)","yes (oldest update 1 year before 18-04-2016)","88","C++, Perl","334","Kraken: ultrafast metagenomic sequence classification using exact alignments"],
["Metavir 2","Roux et al.","19/03/2014","10.1186/1471-2105-15-76","virome analysis","web server for comprehensive virome analysis","too detailed for diagnostics/clinical use(?)","virus","similarity + composition + phylogeny","BLAST, custom, FastTree","nt","RefSeq virus DB, PFAM","-","-","species","no","SPh","yes (webservice)","yes (tables, charts, contig networks)","h - d?","fasta (compressed possible)","short reads or contigs","reads/contigs","HPC","use yes, download no","GNU GPLv3","http://metavir-meb.univ-bpclermont.fr/index.php?page=Tutorial","Any (webservice)","Any","high (webservice)","http://metavir-meb.univ-bpclermont.fr/","-","ViromeScan","-","-","-",", 90% (ViromeScan)","no (problems in 2015 seem unresolved)","24","Perl, Php, Javascript, Css, R, BLAST, FastTree, MetaGeneAnnotator, HMMScan, Uclust, jackhmmer, RaphaelSVG, Cytoscope-web","63","Metavir 2: new tools for viral metagenome comparison and assembled virome analysis"],
["SLIM","Cotten et al.","02/04/2014","10.1371/journal.pone.0093269","reliable detection of viruses from clinical samples and recovery of genomes","iterative mapping and de novo assembly (also in RIEMS)","BLAST (slow and cannot classify everything)","virus","similarity","BLAST","nt","GenBank entries of 2000-500,000 bp long","-","-","species","limited","PAS","no","no","hours of searching, hours for assembly per sample (almost 10x faster than BLAST)","Illumina","150 paired-end","reads","PC","upon request","-","-","Linux, Mac OS","Illumina","low-medium (some dependencies, no free download, changing NR database, �)","-","-","-","-","-","RT-PCR","?","new version in use","14","Python, BLAST, QUASR, MEGAN, BWA, SPAdes, MUMmer","27","Full Genome Virus Detection in Fecal Samples Using Sensitive Nucleic Acid Preparation, Deep Sequencing, and a Novel Iterative Sequence Classification Algorithm"],
["SURPI","Naccache et al.","01/06/2014","10.1101/gr.171934.113","rapid pathogen identification in clinical samples","fast and accurate enough for clinical use","?","pathogens","similarity","RAPSearch2, SNAP","nt, aa","NCBI viral nucleotides & proteins, RefSeq bacterial DB >100 bp sequences, NCBI nt","SNAP","GRCh37/hg19, RefSeq: rRNA, mRNA, mtRNA","species","yes","PFS(AS)","no","yes (coverage maps and pie charts)","~ 1 h (fast), ~ 5 h (comprehensive)","fastq (raw)","75","reads","64 CPU-HPC/Amazon WS","yes","BSD","https://github.com/chiulab/surpi","Linux","Any","low-medium (only tested on Ubuntu 12.04)","https://github.com/chiulab/surpi","BLAST, BWA, Bowtie2, SNAP, RAPSearch","Taxonomer","-","RIEMS","-","84-99%","yes (last commit June 2015)","55","Perl, Pyton, cutadapt, prinseq, gt sequniq, SNAP, fqextract, ABySS, Minimo, RAPSearch2, seqtk. \\Bash, Python, Perl, fastQValidator, Minomo, Abyss, RAPSearch2, seqtk, SNAP, gt-sequniq, fastq, cutadapt, prinseq-lite, dropcache","128","A cloud-compatible bioinformatics pipeline for ultrarapid pathogen identification from next-generation sequencing of clinical samples"],
["Clinical PathoScope","Byrd et al.","01/08/2014","10.1186/1471-2105-15-262","pathogen identification for clinical specimens","few dependencies, rapid, clinically relevant","no assembly, limited to DB","pathogens","similarity","Bowtie2","nt","NCBI bacterial, NCBI viral","Bowtie2","human GRCh37/hg19, GenBank human rRNA","species","limited","SFS","no","no","< 1 h","fastq (pre-processed)","100 to target, 50 to host, 100 to non-target, full read to target","reads","?","yes","GNU GPLv3","https://sourceforge.net/p/pathoscope/wiki/clinical_pathoscope/","Any","Any","medium-high (depends on installation success and database availability)","https://sourceforge.net/projects/pathoscope","RINS, READSCAN, CloVR-metagenomics, IMSA, LMAT, metAMOS","RIEMS","-","-","-","?, 20-100% (RIEMS)","yes (last update: 13-12-2015)","7","Python, Bowtie2","21","Clinical PathoScope: rapid alignment and filtration for accurate pathogen identification in clinical samples using unassembled sequencing data"],
["Exhaustive Iterative Assembly (Virus Discovery Pipeline)","Sch�rch et al.","20/08/2014","10.1371/journal.pone.0105227","virus recovery, discovery and genome assembly","extensive assembly procedure","only viruses, identification still limited to databases","virus","similarity + phylogeny","BLAST, MAFFT, PhyML","nt, aa","NCBI nt, NCBI nr viral fraction, PFAM, custom","BLASTn","GenBank birds, carnivores, rodents, ruminants","species","no","PAFSPh","no","no","interactive","454","252 (simulated)","reads","?","no","-","-","Linux","454","low (settings are provided, but I see no link to the tool)","-","-","-","-","-","-","?","yes (last publication October 2015)","5","Python, Newbler, GSMapper, CAP3, BLAST, HMMER, MEME, MAST, Bioconductor, R","11","Metagenomic Survey for Viruses in Western Arctic Caribou, Alaska, through Iterative Assembly of Taxonomic Units"],
["vFam","Skewes-Cox et al.","20/08/2014","10.1371/journal.pone.0105067","detect viruses in metagenomic data","profile HMMs for viruses","less suited for closely related sequences","virus","similarity","HMMER","aa","RefSeq (virally annotated proteins)","-","-","species","limited","S","no","no","?","fasta","100, 240, 253 nt (averages of 3 datasets)","reads","?","yes","-","-","Linux, Mac OS","Any","high (requires only HMMER, but further instructions seem to be absent)","http://derisilab.ucsf.edu/software/vFam/","BLAST","-","-","-","compared to previous studies","3-73%(?)","no (last modified 12 Feb 2014)","?","HMMER(, CD-HIT, MCL, MUSCLE, BLAST)","19","Profile Hidden Markov Models for the Detection of Viruses within Metagenomic Sequence Data"],
["VirVarSeq","Verbist et al.","26/08/2014","10.1093/bioinformatics/btu587","variant calling for predicting antiviral resistances","codon-level quality control","only well-studied viruses","virus","similarity","BWA","nt","custom (a reference sequence, supplementary n/a (14-02-2017))","-","-","quasispecies","no","SS-pp","no","no","?","Illumina fastq","181 bp fragments in plasmids - supplementary n/a (14-02-2017)","reads","?","yes","-","https://sourceforge.net/projects/virtools/files/?source=navbar","Linux, Mac OS","Illumina","medium","http://sourceforge.net/projects/virtools/?source=directory","LoFreq, V-phaser, ShoRAH","-","-","Posada-Cespedes et al (2016), 10.1016/j.virusres.2016.09.016","-","?","no (last update 29-9-2014)","?","BWA, Q-cpileup, R, Fortran, Perl","13","VirVarSeq: a low-frequency virus variant detection pipeline for Illumina sequencing using adaptive base-calling accuracy filtering"],
["VirFind","Ho & Tzanetakis","01/10/2014","10.1016/j.virol.2014.09.019","virus detection and discovery","automated, universal web-based virus detection and discovery","only web and viruses, limited detection of novel viruses","virus","similarity","BLAST","nt, aa","GenBank nt, GenBank virus proteins","Bowtie2","custom?","species","limited","PFAS(S)","yes (webservice)","no","~ 70 h","sff, fastq, fasta","> 90","reads","64 CPU-HPC","use yes, download no","-","-","Any (webservice)","Any","high (as long as the server is up, web-based and references in article)","http://virfind.org/j/","-","-","-","VIP","-","?","yes (website updated 20-02-2016)","7","fastx-toolkit, seq_crumbs, Bowtie2, Velvet, Blast, CAP3, Python","31","Development of a virus detection and discovery pipeline using next generation sequencing"],
["EnsembleAssembler","Deng et al.","04/01/2015","10.1093/nar/gkv002","assembly of metagenomes","validated metagenome assembly","focus more on contigs than species recovery","metagenomes (especially microbes/viruses)","similarity","Megablast","nt, aa","NCBI RefSeq?","Bowtie2","UCSC/NCBI hg19","species","yes","FPAS","no","no","< 5 minutes (on 8 CPU server)","Illumina",">100","reads","8 CPU-HPC, local should be possible on workstation","yes","-","https://github.com/xutaodeng/EnsembleAssembler","Linux","Any","low-medium (delivers separate tools, with settings to use, rest is to the user)","https://github.com/xutaodeng/EnsembleAssembler","-","VIP","-","-","-","?, 73-99% (VIP, contig coverage)","yes (latest commit 30 November 2014)","11","Python, SoapDenovo, ABySS, MetaVelvet, Cap3, MegaBLAST, Bowtie2, VecScreen","41","An ensemble strategy that significantly improves de novo assembly of microbial genomes from metagenomic next-generation sequencing data"],
["RIEMS","Scheuch et al.","01/03/2015","10.1186/s12859-015-0503-6","sensitive classification of metagenomics datasets","high sensitivity and local installation for confidential samples","long computation time","metagenomes","similarity","GS Mapper, BLAST","nt, aa","NCBI nt/nr","-","-","species","no","PASAS(SS)","no","no","10 h (24 CPU-HPC)","sff, fastq, fasta","no","reads","24 CPU-HPC","yes","GNU GPLv3","-","Linux","454, Ion Torrent, Illumina","low-medium (validation data available, but no instructions)","https://www.fli.de/en/institutes/institut-fuer-virusdiagnostik/labore-arbeitsgruppen/labor-fuer-ngs-und-microarray-diagnostik/","Clinical PathoScope, Kraken","-","-","-","-","83-100%","no (website moved, no date provided)","2","Bash, 454 genome sequencer software suite, Newbler, GS mapper, sff/fna tools, BLAST, Emboss","11","RIEMS: a software pipeline for sensitive and comprehensive taxonomic classification of reads from metagenomics datasets"],
["GOTTCHA","Freitas et al.","12/03/2015","10.1093/nar/gkv180","accurate metagenome classification","fast, low FPs, custom DB","?","bacteria, virus","similarity","BWA (mem)","nt","custom","-","-","species","limited (not tested with real data)","PS-pp","no","no","~ 4 h? (2-5x slower than Kraken)","fastq","fragmented to 30 bp (terminal ends <60 are retained)","reads","16 CPU-HPC, PC","yes","GNU GPLv2","https://github.com/LANL-Bioinformatics/GOTTCHA","Linux, Mac OS","Any","high","https://github.com/LANL-Bioinformatics/GOTTCHA","BLASTn, BWA, MetaPhlAn, mOTUs, Kraken-mini","-","Lindgreen et al. (2016)","-","-","91%, 43% (Lindgreen MCC)","yes (last commit 17 July 2015)","8","Perl, BWA","31","Accurate read-based metagenome characterization using a hierarchical suite of unique signatures"],
["VirSorter","Roux et al.","28/05/2015","10.7717/peerj.985","identification of viral sequences in diverse microbial datasets","focus on microbial data, scoring systems","optimized for bacterial and archaeal viruses, difficulty with short sequences (< 3 kb), and lacks regular prophage features","virus (bacterial)","similarity","BLASTp, HMMER3","aa","Pfam + custom (iPlant Discovery Environment, from RefSeq)","-","-","kingdom","yes","PS","yes (webservice)","no","?","fasta (contigs)","100 bp paired-end reads, assembled by Idba_ud","contigs","?","yes","GNU GPLv2","https://github.com/simroux/VirSorter","Any","Illumina","high","https://github.com/simroux/VirSorter","PhiSpy, Phage_Finder, PHAST","-","-","-","manual curation of prophages","32-100% (short to long contigs)","yes (latest commit 20 October 2016)","?","Perl, HMMER3, MCL, MetaGeneAnnotator, MUSCLE, BLAST","34","VirSorter: mining viral signal from microbial genomic data"],
["Giant Virus Finder","Kerepesi & Grolmusz","14/12/2015","10.1007/s00705-015-2720-8","finding giant viruses in metagenomes","giant viruses","slow, very targeted","virus","similarity","BLAST","nt","NCBI nt + custom reference genomes","-","-","species","no","S","no","no","~30 CPU hours?","anything that BLAST uses?","no","reads","HPC","yes","GNU GPL","http://pitgroup.org/public/giant-virus-finder/latest/README","Linux, Mac OS","Any","medium","http://pitgroup.org/giant-virus-finder","-","-","-","-","-","?","yes","?","Perl, Python, BLAST","3","Giant viruses of the Kutch Desert"],
["ViromeScan","Rampelli et al.","01/03/2016","10.1186/s12864-016-2446-3","virome profiling","fast and unique 2-step filtering","no assembly, no novel viruses","virus","similarity","Bowtie2","nt","NCBI (several custom options)","BMTagger","hg19","species","no","SPFFS","no","yes (tables, graphs)","140 sequences/s/CPU","fastq (compressed possible)","> 60","reads","PC","yes","FreeBSD","https://sourceforge.net/projects/viromescan/files/?source=navbar","Linux, Mac OS","Any","medium-high (depends on installation success)","https://sourceforge.net/projects/viromescan/","Metavir, BLASTn","-","-","-","-","100%","yes","?","Bash, R, Perl, Java, Bowtie2, Bmtagger, Picard","4","ViromeScan: a new tool for metagenomic viral community profiling"],
["GenSeed-HMM","Alves et al.","04/03/2016","10.3389/fmicb.2016.00269","assemble viral genomes for discovery in metagenomic samples, de novo diagnosis, detect emergence of drug-resistance variants, gap closure in assembly finishing projects","progressive assembly with HMM seeds","tested on ssDNA bacteriophage, tested with only a set of assembly tools","any","similarity","BLASTx","aa","reference Microviridae proteins (Roux et al., 2012)","-","-","species","yes","AS","no","no","?","fasta, fastq, sff",">35 bp (101 bp paired-end, 256 bp trimmed)","reads","12 CPU (24 threads) server","yes","GNU GPLv3","https://sourceforge.net/projects/genseedhmm/files","Linux","454, Illumina","medium","https://sourceforge.net/projects/genseedhmm/","Newbler","-","-","-","-","47-100%","yes (last modified 13 Oct 2016)","?","HMMER, BLAST, EMBOSS, CAP3, Velvet, Newbler, SOAPDenovo, ABySS","0","GenSeed-HMM: A Tool for Progressive Assembly Using Profile HMMs as Seeds and its Application in Alpavirinae Viral Discovery from Metagenomic Data"],
["DUDes","Piro et al.","05/03/2016","10.1093/bioinformatics/btw150","metagenomics taxonomic profiling/pathogen detection","novel top-down taxonomy, works with low abundance/partial datasets","still requires quite some manual work","metagenomes/pathogens","similarity","Bowtie2","nt","NCBI GenBank? (DUDesDB)","-","-","species/subtype","limited","S-pp","no","no","15-30 min","fastq?","no","reads","?","yes","-","https://sourceforge.net/projects/dudes/files/README.md/download","Linux","Any","medium-high (depends on installation and databases)","http://sf.net/p/dudes","GOTTCHA, Kraken, MetaPhlAn2","-","-","-","-","87%-95%","yes","?","Python, Bowtie2","3","DUDes: a top-down taxonomic profiler for metagenomics"],
["Unknown pathogens from mixed clinical samples","Gong et al.","17/03/2016","10.1371/journal.pone.0151495","pathogen detection in mixed samples","mixed samples","hard to reproduce: CLCbio","pathogens","similarity + phylogeny","BLAST","nt","NCBI nt","-","-","species","no","PSAPh-pp","no","yes (histogram, tables, coverage map, genetic variants, trees)","interactive","Illumina","no","reads","?","no","-","-","Windows?","Illumina","very low","-","-","-","-","-","PCR, ELISA","?","-","?","CLC Genomics Workbench (7.5), Blast, Bowtie2, Clustal Omega, MEGA6, SimPlot","1","A Next-Generation Sequencing Data Analysis Pipeline for Detecting Unknown Pathogens from Mixed Clinical Samples and Revealing Their Genetic Diversity"],
["VIP","Li et al.","30/03/2016","10.1038/srep23774","virus identification and discovery","fast virus discovery, suitable for desktop use and diagnostics purposes","?","virus","similarity + phylogeny","Bowtie2, RAPSearch2 (depending on mode), MAFFT, ETE","nt, aa (depending on mode)","ViPR + IRD, or RefSeq + neighbors depending on mode, and NCBI human DB","Bowtie2","GRCh38/hg38, RefSeq rRNA, RefSeq, RefSeq mDNA, and bacterial DB from GOTTCHA","subtype","yes","PFSAPh","no","yes (trees, coverage map)","< 2 d?","fast(a/q), (s/b)am","125 (simulated)","reads","PC","yes","GPLv3","https://github.com/keylabivdc/VIP","Linux (Ubuntu, Biolinux)","454, Ion Torrent, Illumina","high (clone from github)","https://github.com/keylabivdc/VIP","EnsembleAssembler, IDBA_UD","-","-","SURPI (","independent confirmatory testing results","?","yes","1","Shell, Python, Perl, PICARD, Bowtie2, MAFFT, Velvet-Oases, RAPSearch2, ETE","5","VIP: an integrated pipeline for metagenomics of virus identification and discovery"],
["Kaiju","Menzel et al.","13/04/2016","10.1038/ncomms11257","fast and sensitive metagenome classification","very fast read classification without huge RAM requirement (up to 6 GB)","only works with CDS, slower with longer sequences","metagenomes","similarity","Kaiju (BWT)","aa","custom protein database?","-","-","species","limited","S","yes/no (webservice/local installation)","no (can use Krona)","~1600-4000 sequences/second/CPU","fasta, fastq","100, 250, 350","reads","HPC/PC (tested on server)","yes","GNU GPLv3","https://github.com/bioinformatics-centre/kaiju","Any (webservice), or Linux, Mac OS","Any","medium (program easy to install, but what database do they use?)","http://kaiju.binf.ku.dk","Kraken, CLARK","-","-","-","-","30-96% (sensitivity@genus-precisions@phylum)","yes","0","C, C++","11","Fast and sensitive taxonomic classification for metagenomics with Kaiju"],
["Taxonomer","Flygare et al.","26/05/2016","10.1186/s13059-016-0969-1","end-to-end universal metagenomics","integrated nucleotide and protein-based classification and simultaneous host mRNA transcript profiling","requires lot of RAM?","pathogens","similarity (exact match k-mers)","KAnalyze + custom k-mer matching","aa","UniRef90 (viruses)","KAnalyze","subsets of Greengenes (bacteria), UNITE (fungi), UniRef50 (bacteria), and ENSMBL human reference sequence","species, strain","yes","FFSS-pp","yes/no (webservice/local installation)","yes","real-time, interactive - < 10 min","fasta, fastq","2x100, 2x250, 139","reads","HPC","yes","MIT","http://taxonomer.iobio.io/instructions.html","Any (webservice), or Linux, Mac OS","Any","high (webserver or clone github)","https://www.taxonomer.com/","Kraken, SURPI, RAPSearch2, DIAMOND","-","-","-","-","~95%","yes","2","Cython, KAnalyze","16","Taxonomer: an interactive metagenomics analysis portal for universal pathogen detection and host mRNA expression profiling"],
["MEGAN Community Edition","Huson et al.","21/06/2016","10.1371/journal.pcbi.1004957","taxonomic and functional profiling","interactive, integrated pipeline with GUI","no viruses tested","metagenomes","similarity","DIAMOND","aa","NCBI NR","-","-","species","limited","S-pp","yes","yes (dendrograms, PCoA plots, tables)","hours per sample on a server (~5.5 h)","Illumina","100 bp paired end","reads","HPC (32 CPU) tested, PC is possible","yes","GNU GPLv3","http://ab.inf.uni-tuebingen.de/data/software/megan6/download/manual.pdf","Any","Illumina","high","http://ab.inf.uni-tuebingen.de/data/software/megan6/download/welcome.html","-","-","-","-","-","?","yes (last commit 16 November 2016)","?","Java, DIAMOND, InterPro2GO, SEED viewer, eggNOG viewer, KEGG","22","MEGAN Community Edition - Interactive Exploration and Analysis of Large-Scale Microbiome Sequencing Data"],
["SMART","Lee et al.","28/07/2016","10.1186/s12859-016-1159-6","classification of complex sequence populations","parallelization with 256 search programs, efficiently match against all species in GenBank","high RAM requirements with multiple CPUs","metagenomes","similarity (256 search programs)","custom","nt","NCBI GenBank (v209)","-","-","species","no","S","no","no","< 10 min or ~2 M reads/min (on HPC - 192 CPUs)","fast(a/q)","no","reads","HPC","for academic use","Academic Licence","https://bitbucket.org/ayl/smart","Linux","Illumina","low-medium (instructions, but dependencies)","https://bitbucket.org/ayl/smart","Kraken, CLARK","-","-","-","-","70-85% (species-class)","yes (May 2016)","?","C++, Ruby, Flash, Sickle, Google SparseHash, GNU parallel","4","Scalable metagenomics alignment research tool (SMART): a scalable, rapid, and complete search heuristic for the classification of metagenomic sequences from complex sequence populations"],
["MetLab","Norling et al.","01/08/2016","10.1371/journal.pone.0160334","provide virologists with an easy to use tool for designing, simulating and analyzing viral metagenomes","complete from study design to analysis, with GUI","not yet for Windows","virus","similarity (exact match k-mers + profile alignment)","Kraken, HMMER3","nt, aa","GenBank viral+phage, RefSeq Bacteria+Archaeae, vFams","Bowtie2","host genome (human)","species","no","P(F)(A)SS","yes","yes (Krona)","< 40 min","raw NGS","no","reads","any UNIX-like","yes","GNU GPLv3","https://github.com/norling/metlab/blob/master/INSTALL.md","Any","Any","medium (few dependencies and documentation available)","https://github.com/norling/metlab","Kraken, Megan (DIAMOND), RAIphy, ProViDE, NBC","-","-","-","-","88-95% (species-kingdom)","yes (June 2016)","?","Python (2.7) + libraries, GNU MPFR, Prinseq-Lite, Bowtie2, SAMTOOLS, SPAdes, Krona Tools, Kraken, FragGeneScan, HMMsearch, vFamParse","0","MetLab: An In Silico Experimental Design, Simulation and Analysis Tool for Viral Metagenomics Studies"],
["metaViC","Modha et al.","24/08/2016","https://www.biostars.org/p/208356/","virus metagenomics for when no host genome is available","needs no host genome to remove host sequences","","virus","similarity","DIAMOND","aa","RefSeq-Protein","RiboPicker","-","species","no","PFSAS","no","yes (Krona)","?","paired-end fastq","no","reads","?","yes","GNU GPLv3","-","Linux, Mac OS","Any","low-medium (all dependencies are mentioned, but there are no instructions)","https://github.com/sejmodha/metaViC","-","-","-","-","-","?","yes (Aug 2016)","?","Bowtie2, DIAMOND, filter_fastq.pl, GARM, IDBA-UD, Kronatools, prinseq, QUAST, riboPicker, SPAdes, Trim Galore","NA","metaViC: Virus metagenomics pipeline for unknown host or in absence of a host genome"],
["VirusSeeker","Zhao et al.","10/01/2017","10.1016/j.virol.2017.01.005","virus discovery and virome composition analysis in a single package","virus discovery and virome composition analysis in a single package","BLAST-based (on HPC)","virus","similarity","BLAST","nt, aa","NCBI viruses (manually curated)","BWA-MEM, MegaBlast, BLASTn, BLASTx","NCBI Bacteria, NCBI NT, NCBI NR (respectively)","species","yes","PSF","no","no","?","paired-end fastq","250","reads","HPC","yes","-","https://wupathlabs.wustl.edu/virusseeker/usage/","Linux","Paired-end Illumina","medium","https://wupathlabs.wustl.edu/virusseeker/","VirFind","-","-","-","-","?","yes","?","Perl, SLURM, BLAST, MegaBLAST, BWA-MEM, cutadapt, ea-utils, PRINSEQ, CD-HIT, Tantan, RepeatMasker, Newbler, Phrap","1","VirusSeeker, a computational pipeline for virus discovery and virome composition analysis"],
["IMSA+A","Cox et al.","19/01/2017","10.1186/s40168-016-0219-5","metatranscriptome classification for bacteria, fungi and viruses in the same sample","efficient and robust classification of different pathogens in the same sample","focus on transcriptome","pathogens","similarity","Bowtie2","nt","the reference genome","-","-","quasispecies","no","FAS","no","no","?","fasta, fastq (all reads are treated as single)","no","reads","?","yes","GNU GPL/CC-ANCSA Intl Public Licence","https://github.com/JeremyCoxBMI/IMSA-A/blob/master/IMSA%2BA_Detailed_Direction.pdf","Linux","Any","medium","https://github.com/JeremyCoxBMI/IMSA-A","MEGAN CE + BLAST/DIAMOND, Kraken","-","-","-","-","95-100% (first taxon level)","yes","?","Python, IMSA, BLAST+, BLAT, Bowtie2, Oases, velvet, Trinity","0","A fast and robust protocol for metataxonomic analysis using RNAseq data"],
["MetaShot","Fosso et al.","27/01/2017","10.1093/bioinformatics/btx036","classification of host-associated microbiome","extract unassigned reads","slower than comparable tools","metagenomes","similarity","Bowtie2, TANGO","nt","NCBI and RefSeq (bacteria, viruses, fungi (from Plantae), invertebrates)","STAR","hg19, GRCh37, 2009","species","no","PFS","no","yes (HTML & Krona)","2-3x slower than Kraken-MetaPhlAn2","paired-end Illumina","no","reads","?","yes","GNU GPLv3","https://github.com/bfosso/MetaShot/blob/bfosso-patch-1/MetaShot%20User%20Guide.pdf","Linux","Paired-end Illumina","medium","https://github.com/bfosso/MetaShot","Kraken, MetaPhlAn2","-","-","-","-","~98%","yes","?","Python, Bash, FaQCs, STAR, Bowtie2, TANGO","0","MetaShot: an accurate workflow for taxon classification of host-associated microbiome from shotgun metagenomic data"]
];
		$(document).ready(function() {
			$('#example').DataTable( {
				data: dataSet,
				columns: [
					{ title: "Name", "defaultContent": "" },
					{ title: "Authors", "defaultContent": "" },
					{ title: "Publication date", "defaultContent": "" },
					{ title: "Doi", "defaultContent": "" },
					{ title: "Goal", "defaultContent": "" },
					{ title: "Unique feature", "defaultContent": "" },
					{ title: "Limitation", "defaultContent": "" },
					{ title: "Target organism(s)", "defaultContent": "" },
					{ title: "Method", "defaultContent": "" },
					{ title: "Search algorithm", "defaultContent": "" },
					{ title: "Sequence type", "defaultContent": "" },
					{ title: "DB (classification)", "defaultContent": "" },
					{ title: "Filter algorithm", "defaultContent": "" },
					{ title: "DB (filter)", "defaultContent": "" },
					{ title: "Classification level", "defaultContent": "" },
					{ title: "Can detect novel/unknown viruses?", "defaultContent": "" },
					{ title: "Steps ", "defaultContent": "" },
					{ title: "GUI?", "defaultContent": "" },
					{ title: "Visualizes results?", "defaultContent": "" },
					{ title: "Runtime", "defaultContent": "" },
					{ title: "Accepts datatypes", "defaultContent": "" },
					{ title: "Tested w/ read length", "defaultContent": "" },
					{ title: "Raw reads/contigs", "defaultContent": "" },
					{ title: "HPC or PC?", "defaultContent": "" },
					{ title: "Freely available?", "defaultContent": "" },
					{ title: "Licence", "defaultContent": "" },
					{ title: "User manual", "defaultContent": "" },
					{ title: "Computer platform", "defaultContent": "" },
					{ title: "Sequencing platform", "defaultContent": "" },
					{ title: "Reproducibility", "defaultContent": "" },
					{ title: "Link (software)", "defaultContent": "" },
					{ title: "Other tools/pipelines tested", "defaultContent": "" },
					{ title: "Tested by (tools)", "defaultContent": "" },
					{ title: "Independent benchmarks", "defaultContent": "" },
					{ title: "Described by", "defaultContent": "" },
					{ title: "Validation", "defaultContent": "" },
					{ title: "Accuracy", "defaultContent": "" },
					{ title: "Up-to-date", "defaultContent": "" },
					{ title: "Citations (Scopus)", "defaultContent": "" },
					{ title: "Software used", "defaultContent": "" },
					{ title: "Citations (Google Scholar 2017-03-28)", "defaultContent": "" },
					{ title: "Title of publication", "defaultContent": "" }
				]
			} );
		} );
	</script>
</body>
</html>
